tests/scenario/test_charm.py

```diff
@@ -0,0 +1,601 @@
+#! /usr/bin/env python
+
+import ops
+import pytest
+import scenario
+import yaml
+from charm import TakahēOperatorCharm
+
+
+@pytest.fixture
+def ctx():
+    # TODO: doesn't scenario have something for this?
+    with open("charmcraft.yaml") as metadata:
+        meta = yaml.safe_load(metadata)
```

Comment:
I was sure it did this automatically, but maybe I'm remembering something else...

Reply:
Yeah, I thought so too, and that it already had support for charmcraft.yaml as well as the separate metadata/actions/config yaml files, but it wasn't working for some reason. I'll look into it again.